src/data/text/move_names.h

```diff
@@ -641,7 +641,7 @@ const u8 gMoveNames[MOVES_COUNT][MOVE_NAME_LENGTH + 1] =
     [MOVE_THROAT_CHOP] = _("Throat Chop"),
     [MOVE_POLLEN_PUFF] = _("Pollen Puff"),
     [MOVE_ANCHOR_SHOT] = _("Anchor Shot"),
-    [MOVE_PSYCHIC_TERRAIN] = _("Psychic Terr"),
+    [MOVE_PSYCHIC_TERRAIN] = _("WeirdTerrain"),
```

Comment:
is it not possible to abbreviate psychic? "Psy. Terrain" maybe? i think i would prefer that to "WeirdTerrain"

Reply:
would also be consistent with Elec Terrain

Reply:
Sure. :)